src/h5web/metadata-viewer/utils.ts

```diff
@@ -38,12 +38,9 @@ export function renderType(type: DType): string {
   return type.class;
 }
 
-export function renderComplex(
-  complex: H5WebComplex | ComplexArray,
-  specifier?: string
-): string {
+export function renderComplex(complex: H5WebComplex | ComplexArray): string {
   if (isH5WebComplex(complex)) {
-    return formatComplex(complex, specifier);
+    return formatScalarComplex(complex);
```

Comment:
`renderComplex` is now only used to render complex attributes in the metadata viewer, so I use the scalar complex formatter (see below).